pkg/acceptance/BUILD.bazel

```diff
@@ -4,7 +4,7 @@ go_library(
     name = "acceptance",
     srcs = [
         "flags.go",
-        "test_main.go",
```

Comment:
Pretty confused by the changes in this file. Is this PR missing some files or something? Otherwise these changes should not be necessary.